docs/KubeSphere/deploy_radondb-mysql_on_kubesphere.md

```diff
@@ -148,79 +148,96 @@ helm package charts
 
 ![应用负载](png/应用负载.png)
 
-  点击 **部署新应用**，在 **来自应用模板**窗口中选择 `radondb-mysql`。
+  点击 **部署新应用**，在 **来自应用模板**窗口中选择 `RadonDB MySQL`。
```

Comment:
From the [picture](https://github.com/runkecheng/radondb-mysql-kubernetes/blob/rename_keyword/docs/KubeSphere/png/%E9%83%A8%E7%BD%B2%E6%96%B0%E5%BA%94%E7%94%A8.png), the application name is `radondb-mysql`